Rajitha/rearrange_UpperCase.js

```diff
@@ -0,0 +1,27 @@
+/*********************************************************************
+* Create a function that moves all capital letters to the front of a word.
+* Examples
+* capToFront("hApPy") ➞ "APhpy"
+* capToFront("moveMENT") ➞ "MENTmove"
+* capToFront("shOrtCAKE") ➞ "OCAKEshrt"
+**********************************************************************/
+const rearrange_UpperCase = str => {
+  let upperStr = "";
+  let lowerStr = "";
+
+  for (let i = 0; i < str.length; i++){ 
+    ((str[i].charCodeAt(0) >= 65) && (str[i].charCodeAt(0) <= 90))? upperStr += str[i] : lowerStr += str[i];
+  }
+  return upperStr.concat(lowerStr);
```

Comment:
will work, but a bit too much. you can just do this `return upperStr  + lowerStr`